### DISCUSSION TITLE
table

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#9311] [Help] [answered]
Hey @melohagan 
Can multiple tables of data be represented in a single table?

**Comment by melohagan** [comment, marked as answer]
If you are using SQL then you can use a JOIN statement in a [custom query](https://docs.budibase.com/docs/custom-queries); otherwise no.

**Reply by rafikApp** [reply]
Thank you @melohagan for your help.
i have another question please.
what does **1-day automation logs** mean?

**Reply by shogunpurple** [reply]
https://docs.budibase.com/docs/logs
